packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -113,22 +132,53 @@ function SelectControl< ItemType = DefaultItemType >( {
 		getItemProps,
 		selectItem,
 		selectedItem: singleSelectedItem,
+		openMenu,
+		closeMenu,
 	} = useCombobox< ItemType | null >( {
 		inputValue,
+		initialIsOpen: isOpen,
+		defaultIsOpen: isOpen,
 		items: filteredItems,
+		selectedItem: null,
```

Comment:
I had to add this, so the `selectedItem` is always passed whenever a item in the list is clicked, this allows us to un-select it right away if keep the selected items as part of the list.
I followed this example: https://www.downshift-js.com/use-combobox#basic-multiple-selection